modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -56,14 +56,21 @@ Example parameter file, `bootstrap-0.parm`, for the bootstrap machine:
 +
 [source,terminal]
 ----
-rd.neednet=1 console=ttysclp0 coreos.inst.install_dev=dasda coreos.live.rootfs_url=http://
-cl1.provide.example.com:8080/assets/rhcos-live-rootfs.s390x.img
-coreos.inst.ignition_url=http://cl1.provide.example.com:8080/ignition/bootstrap.ign
-ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1
-rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0 cio_ignore=all,
-!condev rd.dasd=0.0.3490
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.inst.install_dev=dasda \
+coreos.live.rootfs_url=http://cl1.provide.example.com:8080/assets/rhcos-live-rootfs.s390x.img \
+coreos.inst.ignition_url=http://cl1.provide.example.com:8080/ignition/bootstrap.ign \
+ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1 \
+rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 \
+zfcp.allow_lun_scan=0 cio_ignore=all,!condev \
+rd.dasd=0.0.3490
 ----
-
++
+[NOTE]
+====
+Write all options in parmfile as a single line and make sure you have no newline characters.
```

Comment:
* I've never heard of "parmfile", is that an accurate term? Maybe you could just say "file". It's missing a "the" too, so "in the file"?
* The way this is phrased now kind of sounds like all options have to literally be on a single line, but this isn't exactly the case in the example because you've done the `\` to continue the line. Maybe specify that as an option, that if you want to break them up across multiple lines you must use `\` to continue the options on the next line?

Reply:
`parmfile` is used at least across developers. 
And indeed all options should be on a single line within parmfile, `\` is used here for proper copying (by pressing the Teh symbol on top-right-corner of terminal box) .

Reply:
@nikita-dubrovskii So can users paste that exact text (with the `\` characters) into their .parm file? 

And that's fine about keeping it called parmfile if that's correct, but it still should have a "the" or another article in front of it.

Reply:
> So can users paste that exact text (with the \ characters) into their .parm file?

Yes, if they select text and press Ctrl-C, than Ctrl-V inserts all `\`

Reply:
Re: parmfile. This seems to be an informal way of referring to "parameter file", which is used for [installing on IBM Z](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/performing_a_standard_rhel_installation/index#parameters-and-configuration-files-on-ibm-z_installing-RHEL).